README.tfdoc.hcl

```diff
@@ -0,0 +1,546 @@
+            Port to use to connect with the target. Valid values are either
+            ports `1`-`65535`, or `traffic-port`.
+          END
+        }
+
+        attribute "protocol" {
+          type        = string
+          description = <<-END
+            Protocol to use to connect with the target. 
+          END
+        }
+
+        attribute "timeout" {
+          type        = number
+          description = <<-END
+            Amount of time, in seconds, during which no response means a failed
+            health check. For Application Load Balancers, the range is `2` to
+            `120` seconds, and the default is `5` seconds for the instance
+            target type and `30` seconds for the lambda target type. For
+            Network Load Balancers, you cannot set a custom value, and the
+            default is `10` seconds for TCP and HTTPS health checks and `6`
+            seconds for HTTP health checks.
+          END
+        }
+
+
+
+        attribute "unhealthy_threshold" {
+          type        = number
+          description = <<-END
+            Number of consecutive health check failures required before
+            considering the target unhealthy. For Network Load Balancers, this
+            value must be the same as the `healthy_threshold`. 
+          END
+        }
+      }
+
+      variable "stickiness" {
+        type        = object(stickiness)
+        description = <<-END
+          Stickiness configuration block. 
+        END
+
+        attribute "enabled" {
+          type        = bool
+          default     = true
+          description = <<-END
+            Boolean to enable / disable stickiness.
+          END
+        }
+
+        attribute "type" {
+          type        = string
+          description = <<-END
+            The type of sticky sessions. The only current possible values are
+            `lb_cookie`, `app_cookie` for ALBs, and `source_ip` for NLBs.
+          END
+        }
+
+        attribute "cookie_name" {
+          type        = string
+          description = <<-END
+            Name of the application based cookie. AWSALB, AWSALBAPP, and
+            AWSALBTG prefixes are reserved and cannot be used. Only needed when
+            type is `app_cookie`.
+          END
+        }
+
+        attribute "cookie_duration" {
+          required    = true
+          type        = number
+          description = <<-END
+            Only used when the type is `lb_cookie`. The time period, in seconds,
+            during which requests from a client should be routed to the same
+            target. After this time period expires, the load balancer-generated
+            cookie is considered stale. The range is `1` second to 1 week
+            (`604800` seconds).
+          END
+        }
+      }
+
+      variable "vpc_id" {
+        type        = string
+        description = <<-END
+          Name of the target group. If omitted, Terraform will assign a random,
```

Comment:
Wrong description, this seems to be already wrong in the variable.tf - could you fix this here as well as in the variables? Thank You

Reply:
Done